staging/src/k8s.io/mount-utils/mount_helper_common.go

```diff
@@ -135,3 +143,14 @@ func removePathIfNotMountPoint(mountPath string, mounter Interface, extensiveMou
 	}
 	return notMnt, nil
 }
+
+// removePath attempts to remove the directory. Returns nil if the directory was removed or does not exist.
+func removePath(mountPath string) error {
+	klog.Warningf("Warning: deleting path %q", mountPath)
```

Comment:
this could be noisy, maybe V(4) to match the message "%q is a mountpoint, unmounting"?

Reply:
Ah yes, I'll fix that (it was definitely noisy in my tests)